.editorconfig

```diff
@@ -7,3 +7,9 @@ indent_size = 4
 indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
+
+[*.json]
+indent_size = 2
+
+[.github/**/*.yml]
+indent_size = 2
```

Comment:
not sure what's the point of these changes if PR targets at fixing tests :) only hardens the review

Reply:
Every time I modify the `composer.json` or the `ci.yml` my IDE indents the additions with 4 spaces as defined in the .editorconfig whereas they are indented with 2 spaces, these 2 lines save me from having to correct the indentation manually